vue-frontend/src/components/home-new/BlogSection.vue

```diff
@@ -0,0 +1,256 @@
+<template>
+  <div class="blogs-div">
+    <div class="background">
+      <div class="container">
+        <div class="row">
+          <div class="flex-div">
+            <div class="title-div">
+              <div class="v2-header-3-text">
+                Do we have a team with the right skills?
+              </div>
+              <div class="v2-title-3-text mt-4">
+                Well, at least the community says Hell Yeah. Our blogs hosted on
+                medium have
+                <span class="v2-canopas-gradient-text">100k+</span> minutes
+                monthly reading time and it’s only rising.
+              </div>
+            </div>
+            <img
+              :src="backgroundImage"
+              class="background-image pt-5"
+              loading="lazy"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="container blog-container">
+      <div class="blog-div">
+        <a
+          class="blog-list"
+          v-for="blog in blogs"
+          :key="blog"
+          :href="blog.link"
+          target="_blank"
+        >
+          <aspect-ratio height="50%" class="blog-image">
+            <img :src="blog.thumbnail" class="image" loading="lazy" />
+          </aspect-ratio>
+          <div class="blog-details">
+            <div class="v2-title-3-text title-color">{{ blog.title }}</div>
+            <div class="blog-info mt-4">
+              <div class="v2-normal-3-text title-color blog-date">
+                {{ blog.pubDate }}
+              </div>
+              <div class="v2-normal-3-text title-color blog-author">
+                {{ blog.author }}
+              </div>
+            </div>
+          </div>
+        </a>
+      </div>
+      <div class="view-button">
+        <a class="v2-button v2-normal-2-text" :href="blogsURL" target="_blank">
+          <span>VIEW ALL</span>
+          <font-awesome-icon
+            class="arrow fa"
+            icon="arrow-right"
+            id="leftArrow"
+          />
+        </a>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script type="module">
+import backgroundImage from "@/assets/images/theme/blog-bckground.svg";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import AspectRatio from "@/components/utils/AspectRatio.vue";
+import { mapGetters } from "vuex";
+import store from "@/store";
+import Config from "@/config.js";
+
+export default {
+  data() {
+    return {
+      backgroundImage: backgroundImage,
+      blogsURL: Config.BLOG_URL,
+    };
+  },
+  components: {
+    FontAwesomeIcon,
+    AspectRatio,
+  },
+  computed: {
+    ...mapGetters({
+      blogs: "blogs",
+      blogsError: "blogsErrors",
```

Comment:
What is the use of `blogsError`? Also unified both names `blogsErrors` and `blogsError`